test/features/groups.feature

```diff
@@ -0,0 +1,55 @@
+@users
+Feature: group_show API
+
+    Scenario: Ensure group membership is accessible to sysadmins
+        Given "Admin" as the persona
+        When I log in
+        And I view the "silly-walks" group API "including" users
+        And I take a screenshot
+        Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "group_admin"') and contains(string(), '"name": "walker"')]"
+
+    Scenario: Ensure group membership is accessible to group admins
+        Given "Group Admin" as the persona
+        When I log in
+        And I view the "silly-walks" group API "including" users
+        And I take a screenshot
+        Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "group_admin"') and contains(string(), '"name": "walker"')]"
+
+    Scenario: Ensure group membership is not accessible to non-admin members
+        Given "Walker" as the persona
+        When I log in
+        And I view the "silly-walks" group API "including" users
+        And I take a screenshot
+        Then I should see an element with xpath "//*[contains(string(), '"success": false,') and contains(string(), 'Authorization Error')]"
+
+    Scenario: Ensure group membership is not accessible to admins of other groups
+        Given "Organisation Admin" as the persona
+        When I log in
+        And I view the "silly-walks" group API "including" users
+        And I take a screenshot
+        Then I should see an element with xpath "//*[contains(string(), '"success": false,') and contains(string(), 'Authorization Error')]"
+
+    Scenario: Ensure group membership is not accessible anonymously
+        When I view the "silly-walks" group API "including" users
+        And I take a screenshot
+        Then I should see an element with xpath "//*[contains(string(), '"success": false,') and contains(string(), 'Authorization Error')]"
+
+
```

Comment:
Should this also test the scenarios for sysadmins and group admins accessing group overview?